challenges/frontend.css

```diff
@@ -0,0 +1,250 @@
+body{
+  margin:0;
+  background: linear-gradient(to bottom, #BFCE96 68%, white 60%);
+  background-repeat:no-repeat;
+  font-family: 'Nunito', 'Muli', helvetica;
+  font-weight: normal;
+  color: white
+}
+
+/*navigation bar*/
+.top-nav #logo{
+  top:0;
+  height: 40px;
```

Comment:
I chose px over rem and em for specific elements because I wanted the sizing to be accurate and fixed rather than scalable. However, in the future I would consider using em to make font sizes fully scalable for mobile devices and accessibility purposes or rem for scalability as well as more consistent margins and padding.